src/pages/Video/index.jsx

```diff
@@ -20,13 +21,18 @@ const Video = () => {
   const scrollRef = React.useRef(null)
   const { myVideos, fetchVideos } = useVideos()
   const { videos, isLoading: suggestionsLoading } = myVideos
+  const setDocTitle = useDocumentTitle('Hooli Views')[1]
```

Comment:
Didnt know we can use a single variable from the return value of the custom hook like this. Good one